gef-ui/server/controllers/unissued-facilities/index.js

```diff
@@ -230,13 +230,14 @@ const postChangeUnissuedFacility = async (req, res) => {
     });
 
     const facilityEndDateRedirectUri = `/gef/application-details/${dealId}/unissued-facilities/${facilityId}/facility-end-date`;
+    const bankReviewDateRedirectUri = `/gef/application-details/${dealId}/unissued-facilities/${facilityId}/bank-review-date`;
     const unissuedFacilitiesRedirectUri = `/gef/application-details/${dealId}/unissued-facilities`;
 
-    if (isUsingFacilityEndDate === true) {
-      return res.redirect(facilityEndDateRedirectUri);
+    if (isUsingFacilityEndDate === undefined) {
+      return res.redirect(unissuedFacilitiesRedirectUri);
     }
 
-    return res.redirect(unissuedFacilitiesRedirectUri);
+    return res.redirect(isUsingFacilityEndDate ? facilityEndDateRedirectUri : bankReviewDateRedirectUri);
```

Comment:
same here, this is difficult to read

Reply:
My suggestion above would simplify this 😎